### DISCUSSION TITLE
Getting "undefined" on an empty prompt

### DISCUSSION THREAD

**Discussion by Archie-2021** [SilentVoid13/Templater#808] [Help]
I am getting an undefined as an output whenever I Just put an empty string in something like this: tp.system.prompt("Some text", "")" . I think in the past it was not the case. It is messing with my workflow actually. Is there a way to fix it? Am I misremembering stuff? Should I report it?

**Comment by shabegom** [comment]
Probably related to #801 and should be fixed on next release.
